pkg/apis/kubeone/types.go

```diff
@@ -63,6 +63,15 @@ type KubeOneCluster struct {
 	AssetConfiguration AssetConfiguration `json:"assetConfiguration,omitempty"`
 	// RegistryConfiguration configures how Docker images are pulled from an image registry
 	RegistryConfiguration *RegistryConfiguration `json:"registryConfiguration,omitempty"`
+	// KubeletConfiguration configures the kubelet
+	KubeletConfiguration KubeletConfiguration `json:"kubeletConfiguration"`
+}
+
+// KubeletConfiguration
+type KubeletConfiguration struct {
+	// ContainerLogsMaxSize configure --container-logs-max-size command-line flag of the kubelet. Defaults to 100Mi.
```

Comment:
This comment is not valid any longer because we use the (upstream) KubeletConfig and not command-line flags. You can say something along (copied from [the upstream docs](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)):

```suggestion
	// ContainerLogMaxSize configures the maximum size of container log file before it is rotated
```